### PR TITLE
[BEAM-3397] Fix the failure in the dataflow integration test by removing the spark and flink pipeline options

### PR DESCRIPTION
**Pull request by alanmyrvold**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

Fix the failure in the dataflow integration test by removing the spark and flink pipeline options
Otherwise there are 3 copies of the filesToStage option in the pipelineoptions.

### CODE REVIEW

**Comment by alanmyrvold** [comment]
@lukecwik

File: examples/java/pom.xml [hunk 1]
```diff
@@ -267,6 +267,11 @@
                     <include>WordCountIT.java</include>
                     <include>WindowedWordCountIT.java</include>
                   </includes>
+                  <classpathDependencyExcludes>
```
Comment (kennknowles) [inline examples/java/pom.xml]: Is there a JIRA you can link here about separating the precommit runs and eliminating the `jenkins-precommit` profile?

Comment (alanmyrvold) [reply examples/java/pom.xml]: https://issues.apache.org/jira/browse/BEAM-2596 ?

**Comment by alanmyrvold** [comment]
Does not look like this fixes the issue. Investigating...

**Pull request closed without merging**